Dockerfile

```diff
@@ -1,6 +1,7 @@
 # Build the manager binary
-FROM quay.io/bitnami/golang:1.16 as builder
+FROM golang:1.16 as builder
```

Comment:
This switch back to Docker Hub repository is due to missing architectures in the quay.io hosted one.